cfgov/v1/models/caching.py

```diff
@@ -100,6 +102,21 @@ def purge_all(self):
         self.post_all('invalidate')
 
 
+class AkamaiDeletingBackend(AkamaiBackend):
```

Comment:
The Wagtail frontend cache invalidator API doesn't really support multiple types of purges, where we want to be able to issue both "invalidate" and "delete" purges. The cleanest way to do this and retain use of all the frontend cache invalidator API seems to be introduce a second backend specific to deletion. Because deletion is not something we want to do often, we'll leave it unconfigured. For when we do want to delete, we have a management command that will assemble a configuration for it from the normal Akamai backend.